File: CP/800/1367A.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
	ios::sync_with_stdio(0);
	cin.tie(0);
	
	int t;
	string str;
	
	cin >> t;
	while(t--) {
		cin >> str;
		string ans;
		
		ans += str[0];
		ans += str[1];
		
		for (int i = 3 ; i < (int)str.size(); i += 2) {
			ans += str[i];
		}
		
		cout << ans << "\n";
	}
	
	return 0;
}

File: CP/800/1367A.py
def main():
    t = int(input())

    for i in range(t):
        string = input()
        ans = ""

        i = 0
        while(i < len(string)):
            ans += string[i]

            i += 2

        if (len(string) % 2 == 0):
            ans += string[-1]

        print(ans)


main()
